### PR TITLE
CAT-2051 Duplicate chef task

### PR DESCRIPTION
**Pull request by udeshrl**
@revinjacob Please review and merge

### CODE REVIEW

File: client/cat3/src/partials/sections/dashboard/workzone/orchestration/popups/newTaskCtrl.js [hunk 1]
```diff
@@ -307,6 +326,7 @@
 			$scope.name = "";
 			$scope.taskType = "chef";//default Task type selection;
 			$scope.isEditMode = false;//default edit mode is false;
+            $scope.taskSavig = false;
 			$scope.autoSync = {
```
Comment (revinjacob) [inline client/cat3/src/partials/sections/dashboard/workzone/orchestration/popups/newTaskCtrl.js]: Typo in this line, not in sync with others. Do we need var taskSaving and $scope.taskSaving both?

Comment (udeshrl) [inline client/cat3/src/partials/sections/dashboard/workzone/orchestration/popups/newTaskCtrl.js]: removed this, no longer need this variable

**Comment by revinjacob** [comment]
This can be improved by giving feedback to user by disabling the submit button after first click and waiting for API to return before re-enabling. Such a mechanism is already used in importByIP

**Comment by udeshrl** [comment]
@revinjacob that approach was not working here, getting $apply already in progress error. Had to go this way

**Comment by udeshrl** [comment]
Creating another PR

**Pull request closed without merging**